cmd/sales-api/main.go

```diff
@@ -62,6 +64,11 @@ func main() {
 			SendInterval time.Duration `default:"15s" envconfig:"SEND_INTERVAL"`
 			SendTimeout  time.Duration `default:"500ms" envconfig:"SEND_TIMEOUT"`
 		}
+		Auth struct {
+			KeyID      string `envconfig:"KEY_ID"`
+			PrivateKey string `envconfig:"PRIVATE_KEY"`
```

Comment:
Small note: I like to name config variables like this "PRIVATE_KEY_FILE" b/c it helps me not wonder whether thats a filename or the contents of the key

Reply:
Agreed. I'll change it.